src/common-utils/main.sh

```diff
@@ -471,6 +472,9 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
             cp -rf "${user_rc_file}" "${oh_my_install_dir}" /root
             chown -R ${USERNAME}:${group_name} "${oh_my_install_dir}" "${user_rc_file}"
         fi
+        if [ "$INSTALL_OH_MY_ZSH_CONFIG" != "true" ]; then
+            rm -f "${user_rc_file}" "/root/.zshrc"
```

Comment:
Question: Instead of configuring the `user_rc_file` and then deleting it based on `INSTALL_OH_MY_ZSH_CONFIG`, why not create and configure the file only if `INSTALL_OH_MY_ZSH_CONFIG` is true?

Reply:
Erm, it was just easier for me to create the PR, without moving around any code, its just succinct

Reply:
I was more worried with is the json configuration more cumbersome than the rm command, but suppose its worth in the for UI configuration prompt.

Just need user configured default templates now to auto add this to all my dev-containers.

Reply:
@samruddhikhandale I'm having a trouble: 

- running the tests, you don't know of a way I can run docker build from inside a dev container, reading from the container filesystem? I made a rule that I would run everything inside a virtual machine and keep my main computer free of dependencies.
- The change I have just committed is technically major? Since it not does not override the users zshrc if it already exists, from the base container, or copied from the /etc/skel folder. I can change this if requested? There dons't seem to be a any tests that fail. Just hoping it runs in the github actions.

Reply:
Have you tried using the `devcontainer features test` command? You can run that from within a dev container with docker-in-docker installed. You can simply open this repo with dev containers extension in VS Code or use GitHub codespaces.

Reply:
> The change I have just committed is technically major? Since it not does not override the users zshrc if it already exists, from the base container, or copied from the /etc/skel folder. I can change this if requested?

Yep, added a comment. See https://github.com/devcontainers/features/pull/614#discussion_r1278159617